apps/pg_subscriber/lib/pg_subscriber/handler.ex

```diff
@@ -21,7 +19,8 @@ defmodule PgSubscriber.Handler do
 
   @impl true
   def handle_cast({:handle, message}, state) do
-    handle_message(message)
+    message = handle_message(message)
+    MainApp.Communicator.publish_message(message)
```

Comment:
This feels wrong as it creates circular dependency between `MainApp` and `PgSubscriber`. Is it really ok? Maybe `Communicator` should be a separate app?

Reply:
Trueee.. and I got warning about it, but did not realize this dependency. Very good catch, I will create a new app for that.